.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do we need this file? It is new, but I am not sure what it would be used for

Reply:
I think this might've been me when I clicked debug it created this, will delete.